src/framework/entity/entity_entity.js

```diff
@@ -82,38 +87,44 @@ pc.extend(pc, function () {
         if (system) {
             if (!this.c[type]) {
                 return system.addComponent(this, data);
-            } else {
+            }
```

Comment:
This is probably automatically changed by your text editor but we use brace and else in the same line :D